src/utils/validate.js

```diff
@@ -0,0 +1,74 @@
+import { MESSAGE } from '../constants/message.js'
+
+class validate {
+	inputMoney(inputMoney) {
+		this.#isNotNumber(inputMoney)
+		this.#isZeroOrminus(inputMoney)
+		this.#isNotDivision(inputMoney)
+	}
+	inputWinner(splitNumber) {
+		this.#isNotLengthSix(splitNumber)
+		this.#isNotOverLap(splitNumber)
+		this.#isNotNumberArr(splitNumber)
+	}
+	inputBonus(inputBonusNumber) {
+		this.#isNotNumber(inputBonusNumber)
+		this.#isLottoRange(inputBonusNumber)
+	}
+	randomLottoArr(randomNumber) {
+		this.#isLottoOverlap(randomNumber)
+		this.#isNotLengthSixLotto(randomNumber)
+	}
+
+	#isNotNumber(inputMoney) {
+		if (isNaN(inputMoney)) {
+			throw new Error(MESSAGE.ERROR.NOT_NUMBER)
+		}
+	}
+	#isNotNumberArr(splitNumber) {
+		for (let number of splitNumber) {
```

Comment:
```suggestion
		for (const number of splitNumber) {
```
여기도 `let` 키워드를 사용하신 이유가 있으실까요??

Reply:
위에와 같은 습관입니닿ㅎㅎ